README.md

```diff
@@ -516,8 +393,104 @@ $ make all run
 
 When `< waiting for device >` prompt appears, press reset button
 
-### 6.7 Tips and tricks
-#### 6.7.1 Reference existing project to speed up repo sync
+### 6.7 Juno
+
+Juno has been supported in OP-TEE since mid October 2014.
+
+### 6.7.1 Warnigs
+
+`juno.xml` provides a coherent set of components (OP-TEE client/driver/os,
+Linux kernel version 3-16.0-rc5)
+
++ Further release will align the ARM Juno setup with other OP-TEE supported platforms:
+
+	+ Linux kernel version alignment with QEMU/FVP (DMA_BUF API change).
+	+ Will need arch/arm/Kconfig patch(es) (i.e DMA_SHARED_BUFFER etc...).
+
++ Temporary patch files required for linux kernel and juno dtb definition:
+
+	+ config.linux-linaro-tracking.a226b22057c22b433caafc58eeae6e9b13ac6c8d.patch
+	+ juno.dts.linux-linaro-tracking.a226b22057c22b433caafc58eeae6e9b13ac6c8d.patch
+
+
+#### 6.7.2 Prerequisites
++ The following packages must be installed:
+
+```
+$ sudo apt-get install zlib1g-dev libglib2.0-dev libpixman-1-dev libfdt-dev \
+		       libc6:i386 libstdc++6:i386 libz1:i386 cscope
+```
+
++ Download ARM Juno pre-built binaries:
+
+	+ ARM Juno Pre-built binary bl30.bin (SCP runtime)
+	+ ARM Juno Pre-built binary bl33.bin (UEFI)
+	+ Download at http://community.arm.com/docs/DOC-8401
```

Comment:
This link is not up to date any longer. It refers to https://releases.linaro.org/latest/members/arm/platforms but that link doesn't work. I guess it should have been https://releases.linaro.org/members/arm/platforms/latest instead. Any way I'm not able to find those binaries there.